fabric-chaincode-integration-test/src/test/resources/first-network/scripts/script.sh

```diff
@@ -15,23 +15,13 @@ LANGUAGE="$3"
 TIMEOUT="$4"
 VERBOSE="$5"
 : ${CHANNEL_NAME:="mychannel"}
-: ${DELAY:="3"}
-: ${LANGUAGE:="golang"}
```

Comment:
so this looks like a copy of the golang scripts for first-network? do we need to copy the java equivalent?

Reply:
The scripts are multilanguage - but we don't need that here. so hence the removal,.